src/components/accounts/index.tsx

```diff
@@ -111,6 +122,7 @@ export const Accounts = ({ identities, showAddAccount, next }: AccountsProps) =>
           )}
           label="Add a new account"
           mb={6}
+          {...loadingProps(!!loading)}
```

Comment:
I think when things are loading, we should disable items. Right now it might be the case that a user could still select this even though we are trying to imply that it's disabled. I'd suggest we use pointerEvents: none, and possibly change the cursor to the disabled variant.

Reply:
Yep, should definitely disable pretty much every button on the page while loading